gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -856,74 +983,104 @@ gst_tensor_filter_common_set_property (GstTensorFilterPrivate * priv,
         }
 
         prop->output_meta.num_tensors = num_names;
+      } else if (value) {
+        /** Update the properties */
+        if (GST_TF_FW_V0 (priv->fw)) {
+          /* Once configured, it cannot be changed in runtime */
+          g_assert (!prop->output_configured);
+        } else {
+          /** TODO: update the properties */
+        }
       }
       break;
     case PROP_CUSTOM:
-      /* In case updated custom properties in runtime! */
-      g_free_const (prop->custom_properties);
-      prop->custom_properties = g_value_dup_string (value);
+      if (!priv->prop.fw_opened) {
+        g_free_const (prop->custom_properties);
+        prop->custom_properties = g_value_dup_string (value);
+      } else {
+        /** TODO: update the properties */
+      }
+
       g_debug ("Custom Option = %s\n", prop->custom_properties);
       break;
     case PROP_ACCELERATOR:
     {
-      /**
-       * TODO: allow updating the subplugin accelerator after it has been init
-       * by reopening
-       */
+      gchar *accelerators = g_value_dup_string (value);
       if (priv->prop.fw_opened == TRUE) {
+        /** TODO: update the properties */
+        g_free (accelerators);
         break;
       }
 
-      /** TODO: set this property based on the framework version V0 or V1 */
-      prop->accl_str = g_value_dup_string (value);
+      if (GST_TF_FW_V1 (priv->fw)) {
+        gst_tensor_filter_parse_accelerator (priv, &priv->prop, accelerators);
+        g_free (accelerators);
+      } else {
+        prop->accl_str = accelerators;
+      }
       break;
     }
     case PROP_IS_UPDATABLE:
     {
-      if (priv->fw->reloadModel != NULL)
-        priv->is_updatable = g_value_get_boolean (value);
+      if (GST_TF_FW_V0 (priv->fw) && priv->fw->reloadModel == NULL) {
+        break;
+      } else if (GST_TF_FW_V1 (priv->fw) &&
+          priv->fw->eventHandler (prop, priv->privateData, RELOAD_MODEL, NULL)
```

Comment:
You are not supposed to send "RELOAD" even with this!

Reply:
This call to "RELOAD" is to verify if this event is supported. This call will always fail as the passed parameter is NULL, its just that it shouldn't fail with `-ENOENT`.

Reply:
Then, we need to clearly state that it should return -EINVAL if it supports RELOAD but the given data is NULL (invalid) at the doxygen entry of the API (nnstreamer_plugin_api_filter.h)

Reply:
Added.